### DISCUSSION TITLE
Vim Macro Recording

### DISCUSSION THREAD

**Discussion by DomenPigeon** [JetBrains/ideavim#297] [Ideas]
Hi,
just wanted to ask if it is possible that vim would also record IDE shortcuts or even better autocomplete actions.
When I develop code I am constantly switching between IDE features and VIM features, which allows me to get the best out of the two world, but because macro recording kinda only works for in vim world, it's hard to be productive with it.

If it's possible to do it, it would be awesome :).

**Comment by AlexPl292** [comment]
Hi! Yes, this is one of the big macro issues. Here is the ticket for that, you can vote for it (we mostly prioritize the issues by votes) https://youtrack.jetbrains.com/issue/VIM-913
